marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -747,30 +815,41 @@ const Flows: React.FC<Props> = (props) => {
   );
 
   const flowHeader = (name, index) => (
-    <span id={"flow-header-" + name} className={styles.flowHeader}>
-      <HCTooltip text={props.canWriteFlow ? "Edit Flow" : "Flow Details"} id="open-edit-tooltip" placement="bottom">
-        <span className={styles.flowName} onClick={(e) => OpenEditFlowDialog(e, index)}>
-          {name}
-        </span>
-      </HCTooltip>
-      {latestJobData && latestJobData[name] && latestJobData[name].find(step => step.jobId) ?
-        <HCTooltip text={"Flow Status"} placement="bottom" id="">
-          <span onClick={(e) => OpenFlowJobStatus(e, index, name)} className={styles.infoIcon} data-testid={`${name}-flow-status`}>
-            <ExclamationCircleFill data-icon="exclamation-circle" aria-label="icon: exclamation-circle" className={styles.unSuccessfulRun} />
-          </span>
-        </HCTooltip>
-        : ""
-      }
+    <span><HCTooltip text={props.canWriteFlow ? RunToolTips.flowEdit : RunToolTips.flowDetails} id="open-edit-tooltip" placement="bottom">
+      <span className={styles.flowName} onClick={(e) => OpenEditFlowDialog(e, index)}>
+        {name}
+      </span>
+    </HCTooltip>
     </span>
 
+    /* Commenting out for DHFPROD-7820, remove unfinished run flow epic stories from 5.6, replace above with below later
+    // <span>
+    //   <MLTooltip title={props.canWriteFlow ? "Edit Flow" : "Flow Details"} placement="bottom">
+    //     <span className={styles.flowName} onClick={(e) => OpenEditFlowDialog(e, index)}>
+    //       {name}
+    //     </span>
+    //   </MLTooltip>
+    //   {latestJobData && latestJobData[name] && latestJobData[name].find(step => step.jobId) ?
+    //     <MLTooltip title={"Flow Status"} placement="bottom">
+    //       <span onClick={(e) => OpenFlowJobStatus(e, index, name)} className={styles.infoIcon}>
+    //         <Icon type="info-circle" theme="filled" data-testid={name + "-StatusIcon"} />
+    //       </span>
+    //     </MLTooltip>
+    //     : ""
+    //   }
+    // </span>
+    */
   );
+
+  /* Commenting out for DHFPROD-7820, remove unfinished run flow epic stories from 5.6
```

Comment:
Is there maybe a flag we could use to check the version so if it-s not 5.7 id doesn't show up? Or maybe checking the environment so it doesn't show on prod? I'm thinking that would make it easier than commenting parts of the code, we'll have to remember everything that's commented

Reply:
We could look at the idea, in this case, I cleaned the related code because it is old.